fe/fe-core/src/main/java/org/apache/doris/datasource/CatalogMgr.java

```diff
@@ -744,16 +744,16 @@ public void replayRefreshExternalTable(ExternalObjectLog log) {
         }
     }
 
-    public void dropExternalTable(String dbName, String tableName, String catalogName, boolean ignoreIfExists)
+    public void unloadExternalTable(String dbName, String tableName, String catalogName, boolean ignoreIfExists)
```

Comment:
I think we should unify the name, there is load/unload, register/unregister, create/drop/remove. Too confuse.
I suggest names in this class:

1. register/unregisterExternalTable
2. register/unregisterExternalDatabase